### DISCUSSION TITLE
How to get assessed value for adstock and hill function parameters in meridian.

### DISCUSSION THREAD

**Discussion by chiranjib-acn-cda** [google/meridian#259] [Q&A] [answered]
We wanted to see how delayed impact of media is varying by channel. We wanted to check if we can get the assessed values for alpha (adstock function), ec & slope (hill function) for each channel?

**Comment by yeonjaepark** [comment, marked as answer]
You can check the assessed values for alpha, ec, and slope by channel in the [`inference_data`](https://github.com/google/meridian/blob/60e943ed3b2cabea5e722e5f696c9fb9ccc4025c/meridian/model/model.py#L184-L186) attribute of the Meridian model object.

Here's an example to get the alpha for the media channels:

`mmm.inference_data.posterior.alpha_m`

You can also get the mean and credible intervals for each:

`analyzer.get_mean_and_ci(mmm.inference_data.posterior.ec_m, 0.9)`
